### ISSUE TITLE
voltage regulator: calibrate voltmeter

### ISSUE THREAD

**Issue by heaventwig** [LaboratoryB/camfive#16]
## current functionality 
Voltage output shown on multimeter is 219.5V.
Voltage output shown on voltage regulator is ~150V.

## desired functionality 
Voltage output shown on voltage regulator display matches voltage measured by multimeter.

**Comment by DaBlender** [comment]
Hopefully just the meter. Either through maker or taking it apart to get we can test right before it to isolate. Estimated under$20 part.

**Comment by heaventwig** [comment]
Yeah, with the output being so consistently 219.5V, I suspect this is a meter problem not a thing with the rest of the voltage regulator device.
